dist/stat_data.py

```diff
@@ -825,19 +825,29 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Compression statistics
     ##########################################
-    CompressStat('compress_hist_ratio_2', 'number of blocks with compress ratio smaller than 2'),
-    CompressStat('compress_hist_ratio_4', 'number of blocks with compress ratio smaller than 4'),
-    CompressStat('compress_hist_ratio_8', 'number of blocks with compress ratio smaller than 8'),
-    CompressStat('compress_hist_ratio_16', 'number of blocks with compress ratio smaller than 16'),
-    CompressStat('compress_hist_ratio_32', 'number of blocks with compress ratio smaller than 32'),
-    CompressStat('compress_hist_ratio_64', 'number of blocks with compress ratio smaller than 64'),
-    CompressStat('compress_hist_ratio_max', 'number of blocks with compress ratio greater than 64'),
     CompressStat('compress_precomp_intl_max_page_size', 'compressed page maximum internal page size prior to compression', 'no_clear,no_scale,size'),
     CompressStat('compress_precomp_leaf_max_page_size', 'compressed page maximum leaf page size prior to compression ', 'no_clear,no_scale,size'),
-    CompressStat('compress_read', 'compressed pages read'),
-    CompressStat('compress_write', 'compressed pages written'),
-    CompressStat('compress_write_fail', 'page written failed to compress'),
-    CompressStat('compress_write_too_small', 'page written was too small to compress'),
+    CompressStat('compress_read', 'compressed pages read from disk'),
+    # dist/stat.py sorts stats by their descriptions and not their names. The following stat descriptions insert an extra 
+    # space before the single digit numbers (2, 4, 8) so stats will be sorted numerically (2, 4, 8, 16, 32) instead of 
+    # alphabetically (16, 2, 32, 4, 8).
+    CompressStat('compress_read_ratio_hist_2', 'compressed pages read from disk with compression ratio smaller than  2'),
```

Comment:
I don't remember, but does the "CompressStat" already add a standard prefix? i.e: Will the word compress appear three times in the output with the proposed implementation? If not, could the prefix be pulled up there..

If that isn't an option, I'd vote for changing the last two stats so they share a common prefix and will sort together in t2.

Reply:
Yep it does. Example t2 statistic: `wt table compression compressed pages read`
I'll drop the first use of compression so stats will read as
`wt table compression pages read from disk with compression ratio smaller than  2`

Reply:
Updated in [b575a62](https://github.com/wiredtiger/wiredtiger/pull/9908/commits/b575a62ad44bb6b3ca6992d30b7d0c6eef858b61)